itest/lnd_sweep_test.go

```diff
@@ -24,126 +24,163 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// testSweepAnchorCPFPLocalForceClose checks when a channel is force closed by
-// a local node with a time-sensitive HTLC, the anchor output is used for
-// CPFPing the force close tx.
+// testSweepCPFPAnchorOutgoingTimeout checks when a channel is force closed by
+// a local node due to the outgoing HTLC times out, the anchor output is used
+// for CPFPing the force close tx.
 //
 // Setup:
-//  1. Fund Alice with 2 UTXOs - she will need two to sweep her anchors from
-//     the local and remote commitments, with one of them being invalid.
-//  2. Fund Bob with no UTXOs - his sweeping txns don't need wallet utxos as he
-//     doesn't need to sweep any time-sensitive outputs.
-//  3. Alice opens a channel with Bob, and sends him an HTLC without being
-//     settled - we achieve this by letting Bob hold the preimage, which means
-//     he will consider his incoming HTLC has no preimage.
-//  4. Alice force closes the channel.
+//  1. Fund Alice with 1 UTXOs - she only needs one for the funding process,
+//  2. Fund Bob with 2 UTXOs - he needs one for the funding process, one for
```

Comment:
Yeah it's fixed! Here's what happened, since we are creating two sweeping txns for each of the anchors,
- if remote anchor gets created first -> `testmempoolaccept` says no -> the wallet utxo is unlocked -> the same wallet utxo is now used for the local anchor sweep.
- if the local anchor gets created first -> `testmempoolaccept` says yes -> the wallet utxo is locked -> when creating the remote anchor it will return `ErrNotEnoughInputs`.